corehq/apps/users/static/users/js/roles_and_permissions.js

```diff
@@ -0,0 +1,30 @@
+hqDefine("users/js/roles_and_permissions",[
+    'jquery',
+    'knockout',
+    'underscore',
+    "hqwebapp/js/initial_page_data",
+    'users/js/roles',
+    'bootstrap', // for bootstrap modal
```

Comment:
Shouldn't need this because it's included in [common.js](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/hqwebapp/static/hqwebapp/js/common.js) and you aren't directly accessing it.

Reply:
removed